geniza/corpus/models.py

```diff
@@ -2,7 +2,7 @@
 
 from django.db import models
 from django.urls import reverse
-from django.db.models.functions import Concat
+from django.db.models.functions import Concat, Coalesce
```

Comment:
does `Coalesce` get used?

Reply:
good catch, thanks — I tried it but it didn't help, guess I forgot to remove!